cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -690,6 +690,37 @@ <h2>Rewards and perks</h2>
             </dd>
         {% endif %}
     </dl>
+    <h2>Contact information</h2>
+    <dl class="m-list m-list__card-details">
+        {% if card.website_for_consumer %}
+            <dt>Website</dt>
+            <dd class="a-card-url">
+                {# Some issuers submitted more than one URL for a card. In all
+                    of those instances, the URLs are separated with a space, so
+                    we'll only turn the submitted URL into a link if it doesn't
+                    have a space in it for now. TODO: maybe split multiple URLs
+                    up.
+                #}
+                {% if ' ' is not in card.website_for_consumer %}
+                    {# TODO: Do we need to run this through link middelware? #}
+                    <a href="{{ card.website_for_consumer }}">
+                        {{ card.website_for_consumer }}
+                    </a>
```

Comment:
The middleware will already process this link, so there's no additional step needed.

```suggestion
                    <a href="{{ card.website_for_consumer }}">
                        {{- card.website_for_consumer -}}
                    </a>
```

We should remove the whitespace between the `<a> text </a>`, though, otherwise the underline gets extended improperly like this:

<img width="76" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/098d08bf-bff0-49ff-b201-8feae52c37eb">

Reply:
Fixed in c087f88